test/e2e/util_gatewayapi_test.go

```diff
@@ -562,14 +563,14 @@ func assertGatewaySuccessful(t *testing.T, namespace, name string) (*gatewayapiv
 
 	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 1*time.Minute, false, func(context context.Context) (bool, error) {
 		if err := kclient.Get(context, nsName, gw); err != nil {
-			t.Logf("failed to get gateway %s, retrying...", name)
+			t.Logf("Failed to get gateway %v: %v", nsName, err)
```

Comment:
nit:
```suggestion
			t.Logf("Failed to get gateway %v: %v, retrying...", nsName, err)
```

Reply:
Done in https://github.com/openshift/cluster-ingress-operator/compare/77beec1a8b6af8c8c5ce00f29f7a60683a84cd75..92e89a2a02fccb3d149e3e0491a30f19fe160774.